src/app/(admin)/admin/layout.tsx

```diff
@@ -29,11 +30,14 @@ export default function AdminLayoutPage({
     return () => unsubscribe();
   }, [user, setUser]);
 
+  if (!session) return <LoadingScreen />;
```

Comment:
i would return null just to make it less obvious that there's an admin panel here... otherwise perfect